### PR TITLE
Add rcsplit support to inav

### PR DESCRIPTION
**Pull request by azolyoung**
Hi there,

Here is azol, I'm adding rcsplit control to iNav FC, this pull request is about the configurator that support rcsplit.

### CODE REVIEW

File: tabs/ports.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ TABS.ports = {};
 
 TABS.ports.initialize = function (callback, scrollPosition) {
     var board_definition = {};
+    var isSupportPeripherals = semver.gte(CONFIG.apiVersion, "1.27.0");
```
Comment (digitalentity) [inline tabs/ports.js]: Actual MSP API didn't change. I think it's better to check for firmware version, not for apiVersion. @DzikuVx what do you think?

Comment (azolyoung) [inline tabs/ports.js]: @digitalentity 
I've update it, and I'm post a  wrong comment of the last commit in this PR, in my code, I'm using FC version to detect which is support peripherals configuration or not.
But actually, the result is same.

Comment (digitalentity) [inline tabs/ports.js]: Great! I need to test this

File: tabs/ports.html [hunk 2]
```diff
@@ -62,6 +63,9 @@
                 <td class="functionsCell-gps"><select class="gps_baudrate">
                         <!-- list generated here -->
                 </select></td>
+                <td class="functionsCell-peripherals"><select class="blackbox_baudrate">
```
Comment (digitalentity) [inline tabs/ports.html]: Do I understand correctly that it would be using the same baud rate as selected for blackbox?

**Pull request merged**